lib/Reader/reader.cpp

```diff
@@ -5211,6 +5211,8 @@ ReaderBase::rdrCall(ReaderCallTargetData *Data, ReaderBaseNS::CallOpcode Opcode,
 
   std::vector<CallArgType> &ArgumentTypes =
       Data->CallTargetSignature.ArgumentTypes;
+
+  // Vector is being initialized for the first time as it was skipped in init.
```

Comment:
Data->CallTargetSignature is created using _alloca and its constructor is never called. Therefore Data->CallTargetSignature.ArgumentTypes has not been constructed when we get to this point. The comment was meant to explain that the placement new is necessary here and .resize would cause a AV.